.github/workflows/python_safety.yml

```diff
@@ -8,9 +8,12 @@ jobs:
     runs-on: 'ubuntu-20.04'
     steps:
       - run: sudo apt-get -q update
-      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends dist-upgrade
       - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends install
              curl gcc ffmpeg libcurl4-openssl-dev libssl-dev motion v4l-utils
       - uses: actions/checkout@v4
-      - run: pip install --user safety .
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install --upgrade pip setuptools
```

Comment:
Probably with the `-U` at least all dependencies are installed with max possible versions?
```suggestion
      - run: pip install --upgrade pip setuptools
      - run: pip install -U .
```

Reply:
We want to install `safety` and `.` in a normal way like a normal user would without forcing to latest and greatest.

Reply:
Ah, I missed the `.` in the command below. Okay the test is green already, otherwise I guess we'd need to raise the minimum dependency versions to assure no vulnerable version can be installed.